mpc-core/src/ot/extension/mod.rs

```diff
@@ -229,6 +297,15 @@ mod tests {
         assert!(sender.is_complete());
         assert!(receiver.is_complete());
 
+        // Trying to send/receive more OTs should return an error
+        let err = sender.send(&[[Block::random(&mut rng); 2]]).err().unwrap();
```

Comment:
The array expression here probably doesn't do what you think it does. It creates a random `Block` X and makes an array `[X, X]`. You probably want `[X, Y]` for a uniform Y.